x-pack/test/alerting_api_integration/common/plugins/task_manager_fixture/server/plugin.ts

```diff
@@ -95,6 +95,34 @@ export class SampleTaskManagerFixturePlugin
         return res.ok({ body: {} });
       }
     );
+
+    router.post(
```

Comment:
all the functional tests that test this functionality load data from an ES archive. doing this overwrites the task manager index so we have to ensure this task is scheduled before scheduling it to run.